packages/sdk/src/utils/deterministicGuids.ts

```diff
@@ -5,6 +5,10 @@
 
 import * as crypto from 'crypto';
 
+// tslint:disable-next-line:variable-name
+const ZeroGuid = '00000000-0000-0000-0000-000000000000';
+export { ZeroGuid };

```

Comment:
prefix with export and make it a one liner.

Reply:
This can be deleted now correct?

---
In reply to: [253243773](https://github.com/Microsoft/mixed-reality-extension-sdk/pull/164#discussion_r253243773) [](ancestors = 253243773)

Reply:
Oh I see you backed it out.

---
In reply to: [253566140](https://github.com/Microsoft/mixed-reality-extension-sdk/pull/164#discussion_r253566140) [](ancestors = 253566140,253243773)